package.json

```diff
@@ -256,6 +276,21 @@
                 "command": "aws.showErrorDetails",
                 "title": "%AWS.command.showErrorDetails%",
                 "category": "AWS"
+            },
+            {
+                "command": "aws.viewLogs",
```

Comment:
Is `aws.viewLogs` from a different PR? I'm wondering if it should be removed from this one.

Reply:
Yes, I'm actually not sure how it got in there (I made this branch off of a separate pull from `deploy`. It's from the logging PR: https://github.com/aws/aws-toolkit-vscode/pull/302